contents/handbook/company/security.md

```diff
@@ -10,40 +10,42 @@ It is critical that everyone in the PostHog team follows these guidelines. We ta
 
 ### Policies
 
-We have a number of policies in place to support SOC 2 compliance. All team members have been invited to Drata to review these and to complete security training as part of onboarding. Additionally, all team members undergo background checks when starting at PostHog. 
-
-Our policies are stored on Drata, so this section of the Handbook just serves to make these policies publicly available for potential customers:
-
-- [Acceptable Use Policy](https://drive.google.com/file/d/1rZknmogF7B3KVxWqOAl278bUckWDv92v/view?usp=sharing)
-- [Asset Management Policy](https://drive.google.com/file/d/14Z3Isvffwg7Y8X_ZHMvp_7gdvDujAwRe/view?usp=sharing)
-- [Backup Policy](https://drive.google.com/file/d/19fKnj20U48rmkDPpes2wA_hK7hYH45aP/view?usp=sharing)
-- [Business Continuity Plan](https://drive.google.com/file/d/1sJWpNr9U2aONrKOJyrbgKqxDi-h8Z3hx/view?usp=sharing)
-- [Code of conduct](https://drive.google.com/file/d/1qVtkxDBmKCMFUX3cgichKEsW0IylliQG/view?usp=sharing)
-- [Data Classification Policy](https://drive.google.com/file/d/1VFoba8mrDiTHo0A0po1hLWWZ5gsfTSM6/view?usp=sharing)
-- [Data Deletion Policy](https://drive.google.com/file/d/1FBNFBC0lZHi6VE1z-PYjlYHzERIkKBmX/view?usp=sharing)
-- [Data Protection Policy](https://drive.google.com/file/d/1C-P0QxxaayEHgOdoUEiSvwY_VwriaFHo/view?usp=sharing)
-- [Disaster Recovery Plan](https://drive.google.com/file/d/1VGOGvRE22NDsN0SV32ZtG27gNLJWVqbN/view?usp=sharing)
-- [Encryption Policy](https://drive.google.com/file/d/1mX9s8gRRpOs7UpdZ48KGyCErL8a3DfZD/view?usp=sharing)
-- [Incident Response Plan](https://drive.google.com/file/d/1Dnj_gELBJTMlqTagGs3Mv8JWMbaLXM54/view?usp=sharing)
-- [Information Security Policy](https://drive.google.com/file/d/1Z2S-yl0jBPLVdl_0Qwc5kyIF3Dj3ypTp/view?usp=sharing)
-- [Password Policy](https://drive.google.com/file/d/1FlOKt_yHGSH8vKMsm4z04M58RiyrWo4M/view?usp=sharing?usp=sharing)
-- [Physical Security Policy](https://drive.google.com/file/d/17JkSfMC7ILuAm3YjZRCTB7i8eWLtkuN3/view?usp=sharing)
-- [Responsible Disclosure Policy](https://drive.google.com/file/d/1ag8F2OA3FYUwRRAGbzMrDw1XV1QoqhNg/view?usp=sharing)
-- [Risk Assessment Policy](https://drive.google.com/file/d/1mnqKDqZTjOI4EJhpbpbjcVmLzCMhZDki/view?usp=sharing)
-- [Software Development Lifecycle Policy](https://drive.google.com/file/d/1FU8quDWJi66bJnAKc-9ZPPfL7Skp07n9/view?usp=sharing)
-- [System Access Control Policy](https://drive.google.com/file/d/1jxiy9OpS4aCllDQJk88emnihWKUv6Uyq/view?usp=sharing)
-- [Vendor Management Policy](https://drive.google.com/file/d/1AQxJ9k4V6kXzECdyS2Fn5h-BcXrgQQDi/view?usp=sharing)
-- [Vulnerability Management Policy](https://drive.google.com/file/d/1bUxuBvTCAzMasG39ShtfnUjRm_2gM1Q3/view?usp=sharing)
+We have a number of policies in place to support SOC 2 compliance. All team members have been invited to Drata to review these and to complete security training and background checks as part of onboarding.
```

Comment:
@charlescook-ph Any reason we're hiding these now?  Its been really useful for me in security reviews to just forward them this handbook page.

Reply:
Most of these got updated when Daniel and Grace were doing their prep work over the last couple of months, so they're out of date (and I couldn't be bothered to download all the policies, add to Google Drive and re-link...)